regparser/web/jobs/utils.py

```diff
@@ -0,0 +1,91 @@
+from regparser.tasks import run_eregs_command
+import django_rq
+import settings
+
+
+def queue_eregs_job(args, timeout=60*30, result_ttl=-1):
```

Comment:
Heads up that a `result_ttl` of `-1` (i.e. never delete) is okay for demonstration purposes, but we'll want to change it before too long. `rq` more or less has a list associated with a key like `rq:active_jobs` that'll keep just keep growing; this list'll be returned for _every_ `queue` interaction in the future.